tests/OSPSuite.Starter/Tasks/ImporterConfigurationDataGenerator.cs

```diff
@@ -130,9 +131,9 @@ public IReadOnlyList<MetaDataCategory> DefaultGroupByTestMetaDataCategories()
          return categories;
       }
 
-      public IReadOnlyList<ColumnInfo> DefaultGroupByConcentrationImportConfiguration()
+      public Cache<string, ColumnInfo> DefaultGroupByConcentrationImportConfiguration()
       {
-         var columns = new List<ColumnInfo>();
+         var columns = new Cache<string, ColumnInfo>(getKey: x => x.DisplayName);
```

Comment:
Why do we have name and display name in columnInfo. I am getting a bit confused

Reply:
I think they are not both needed since we always have them assigned to the same value. So we could create an issue to remove one of them (DisplayName I would say)